pwa/src/templates/templateParts/footer/FooterTemplate.tsx

```diff
@@ -97,7 +97,7 @@ export const FooterTemplate: React.FC<FooterTemplateProps> = ({ layoutClassName
 
             <li
               onClick={() => {
-                navigate("#");
+                navigate("/apiDocs");
```

Comment:
Routes are usually snake cased, not camel.

So in this case that'd become `api-docs` instead of `apiDocs`.

Besides that, I think it would be better to create a subroute `documentation`.

So this page could land on `/documentation/api`.